### ISSUE TITLE
Unevenly Thick and Spaced Bars in Horizontal Bar Plot

### ISSUE THREAD

**Issue by mprt** [piccolomo/plotext#78]
Hi,
I'm trying to make a nice looking horizontal bar-plot, but the bars are always unevenly shaped.
I've been playing with plot_size and width, but some things are always off:
- different spaces between the bars
- different bar widths
- different position of bars with respect to their ticks.

I'd appreciate some pointers, thanks! :)

**Comment by piccolomo** [comment]
Hello @mprt ,

thanks for the report and happy to help? Could you give me some minimal code, so that i tried to fix it?

**Comment by mprt** [comment]
Hi  @piccolomo , 
thanks for the quick reply!
here's a quick example
```
import plotext as plt

plt.clc()
y = [str(x) for x in list(range(16))]
x = [n * 3 for n in list(range(16))]

plt.bar(y, x, orientation = "h",  width=0.1, minimum=0)
plt.xlim(0, 50)
plt.plot_size(width=None, height=22)
plt.show()
```
Here's how it looks on my machine.
https://ibb.co/wQs7hG2

I guess it's just important to get the "fractions" of width and plot_size right, but I haven't found a solution yet.

Thanks for your help!

**Comment by piccolomo** [comment]
Solved:

```python
import plotext as plt
plt.clf()
plt.clc()
y = [str(x) for x in list(range(16))]
x = [n * 3 for n in list(range(16))]
plt.bar(y, x, orientation = "h",  width=0.3, minimum=0, marker = "hd")# works also with marker="fhd"
plt.xlim(0, 50)
plt.plotsize(None, 2 * len(x) - 1 + 3) 
plt.show()
```
I followed my own guide [here](https://github.com/piccolomo/plotext/blob/master/readme/bar.md#sketchy-bar-plot) where, instead of `2 * len(x) - 1 + 3`, I plugged `2 * len(x) - 1 + 3` because there is no title (if you add a title use 4 then).

It doesn't work with marker = "sd" and i am not sure why, but this is a different story. Also I found the value width=0.3 by chance, I would need to think about why that value works. 

Anyway I think it solves the issue. Let me know otherwise. :-)

All the best,
Savino
